### PR TITLE
Add Hydrawise

### PR DESCRIPTION
**Pull request by mobilutz**

### CODE REVIEW

File: _data/other.yml [hunk 1]
```diff
@@ -261,6 +261,13 @@ websites:
       exceptions:
           text: "2FA only for North American customers that create Personal campaigns."
 
+    - name: Hydrawise
+      url: https://www.hydrawise.com/
+      img: hydrawise.png
+      fta: No
```
Comment (mxxcon) [inline _data/other.yml]: it's `tfa`, not `fta`.

**Review by mxxcon — CHANGES_REQUESTED** [review]

**Comment by mobilutz** [comment]
@mxxcon Thanks for the hint. I changed it.

File: _data/other.yml [hunk 2]
```diff
@@ -261,6 +261,13 @@ websites:
       exceptions:
           text: "2FA only for North American customers that create Personal campaigns."
 
+    - name: Hydrawise
+      url: https://www.hydrawise.com/
+      img: hydrawise.png
+      tfa: No
+      tritter: Hunter_Ind
```
Comment (mxxcon) [inline _data/other.yml]: This should be `twitter`, not tritter. didn't notice it the first time, but spelling is important ;)

**Review by mxxcon — CHANGES_REQUESTED** [review]

**Comment by mobilutz** [comment]
@mxxcon So sorry about these typos. It looks like I was not really paying attention.
Thanks for your feedback

**Comment by brycx** [comment]
Thanks for the PR @mobilutz. We currently have a limitation on new additions, that require a site to be within the Alexa top 200k, which Hydrawise [is sadly not](https://www.alexa.com/siteinfo/hydrawise.com). It's rising though, so maybe we can add it later.

**Comment by mxxcon** [comment]
Hmm. when I was looking at it it had a higher alexa ranking 🤔

**Comment by mobilutz** [comment]
@mxxcon I think that @brycx mean global rank.
screen shot 2018-08-06 at 16 06 25

**Pull request closed without merging**